README.md

```diff
@@ -28,11 +28,20 @@ Detailed information on features and capabilities can be found in the [Modulus d
 
 ## Modulus Packages
 
-- [Modulus (Beta)](https://github.com/NVIDIA/modulus)
-- [Modulus Launch (Beta)](https://github.com/NVIDIA/modulus-launch)
-- [Modulus Symbolic (Beta)](https://github.com/NVIDIA/modulus-sym)
-- [Modulus Tool-Chain (Beta)](https://github.com/NVIDIA/modulus-toolchain)
-
+- [Modulus (Beta)](https://github.com/NVIDIA/modulus): Open-source deep-learning
+  framework for building, training, and fine-tuning deep learning models using
+  state-of-the-art Physics-ML methods.
+- [Modulus Symbolic (Beta)](https://github.com/NVIDIA/modulus-sym): Framework
+  providing pythonic APIs, algorithms and utilities to be used with Modulus
+  core to physics inform model training as well as higher level abstraction
+  for domain experts.
+
+### Domain Specific Packages
+
+- [Earth-2 MIP (Alpha)](https://github.com/NVIDIA/earth2mip): Python framework
```

Comment:
Change this to Beta, I need to update the read me.

Reply:
Done!